dockerfiles/mpi-worker-start.sh

```diff
@@ -47,6 +48,20 @@ for gid in `id -G`; do
   fi
 done
 
+cat << EOF > "$DOMINO_HOME_DIR/.profile"
+if [ "\$BASH" ] && [ -f ~/.bashrc ]; then
+  . ~/.bashrc
+fi
+EOF
+chmod 644 "$DOMINO_HOME_DIR/.profile"
+
+rm -f "$DOMINO_HOME_DIR/.bashrc"
+touch "$DOMINO_HOME_DIR/.bashrc"
+printenv | grep PATH | sed 's;^;export ;' >> "$DOMINO_HOME_DIR/.bashrc"
+printenv | grep MPI | sed 's;^;export ;' >> "$DOMINO_HOME_DIR/.bashrc"
+printenv | grep DOMINO | sed 's;^;export ;' >> "$DOMINO_HOME_DIR/.bashrc"
+chmod 644 "$DOMINO_HOME_DIR/.bashrc"
```

Comment:
This is read-only by the `domino` user right?

Reply:
Oops. The `.profile` and `.bashrc` should be owned by domino. Will fix now. A mode probably doesn't matter, and I set to to 644 only because it's a usual mode for this kind of files.